symbolic/src/Data/Macaw/Symbolic/Testing.hs

```diff
@@ -198,6 +238,13 @@ functionName dfi = maybe addrName fromByteString (MD.discoveredFunSymbol dfi)
 -- *translation* of programs into Crucible, while 'MS.MacawArchEvalFn' is used
 -- during symbolic execution (to provide interpretations to
 -- architecture-specific primitives).
+--
+-- In addition to symbolically executing the function to produce a Sat/Unsat
+-- result, this function will attempt to verify all generate side conditions if
+-- the name of the function being simulated begins with @test_and_verify_@ (as
+-- opposed to just @test@).  This is necessary for testing aspects of the
```

Comment:
It would be good to make a note of this convention in the `README`s in each of the test suites' top-level directories (`macaw-aarch32-symbolic/tests/README.org`, `macaw-ppc-symbolic/tests/README.org`, and `x86_symbolic/tests/README.org`).